mimesis/providers/path.py

```diff
@@ -9,10 +8,10 @@
 class Path(BaseProvider):
     """Class that provides methods and property for generate paths."""
 
-    def __init__(self, *args, **kwargs):
-        # TODO: platform should be a parameter
+    def __init__(self, platform=sys.platform, *args, **kwargs):
```

Comment:
This `platform`s should be defined as constants in https://github.com/lk-geimfari/mimesis/tree/master/mimesis/constants.